src/browser/web-viewer.js

```diff
@@ -63,6 +64,9 @@ define((require, exports, module) => {
   WebViewer.onLoadStart = webViewerCursor => event => webViewerCursor.merge({
     readyState: 'loading',
     isLoading: true,
+    connecting: true,
+    startLoadingTime: performance.now(),
```

Comment:
We need to use one consistent way to measure time. `timeStamp` is available from events. We can't use this value with `performance.now()` (from RFA).